root/etc/cb/integrations/infoblox/infoblox.conf.example

```diff
@@ -89,9 +90,34 @@ carbonblack_streaming_host=192.168.230.206
 carbonblack_streaming_username=cb
 carbonblack_streaming_password=FiDk1Fz5aFOk1Oj7
```

Comment:
This isn't a token that is used anymore, but go ahead and XXXX it out just to make clear it's an example